modules/currency-l0/src/main/scala/org/tessellation/currency/l0/modules/Storages.scala

```diff
@@ -53,7 +52,6 @@ object Storages {
         )
       lastGlobalSnapshotStorage <- LastSnapshotStorage.make[F, GlobalIncrementalSnapshot, GlobalSnapshotInfo]
       globalL0ClusterStorage <- L0ClusterStorage.make[F](globalL0Peer)
-      lastBinaryHashStorage <- LastBinaryHashStorage.make[F]
```

Comment:
This storage isn't needed anymore. Last binary hash gets passed around as part of consensus outcome.